### ISSUE TITLE
Should the multi pitch carry more weight than single pitch

### ISSUE THREAD

**Issue by scd** [theCrag/website#2703]
I recall reading somewhere in the explanations that you would value some feedback. My thoughts are that multipitch routes should carry more weight as opposed to single pitch routes. I'm not at all sure how you convert that into a meaningful value.

As I understand that the current system would have it, a multipitch and single pitch are counted equally. It goes without saying that 10 pitches of 18 is harder than a single pitch of 18, but the system counts the multipitch as one route, not as 10 pitches. My thoughts are that a fairer rating would in some way accomodate for this added commitment and sustained athletic performance.

**Comment by scd** [comment]
There is possible merit to this, however we would have to first implement pitches properly and assign a cpr for each pitch grade. Different score for seconding versus leading each pitch.

**Comment by brendanheywood** [comment]
Yeah it's a tricky one, because it's not just pitches but also height as well. Some grade systems reflect the hardest move, others take into consideration how sustained it is. You could have a 100m multipitch which is 5 pitches and another similar route of 100m which is 3 pitches, how much do the rests at the belay factor in? If any thing the 3 pitches would be harder than the 5 pitches. You could have a 20m climb with a single grade 21 crux, or another which is fairly sustained at 21 the whole way.

We also have to be careful that we don't blow out the power factor, for instance if we simplistically counted every 20m of grade X as a pseudo tick, then a 100m grade X climb would be 5 ticks and be enough to bump you up, which doesn't feel right to me.

**Comment by scd** [comment]
If the pitches have grades 18,16,22 then the final CPR will be pretty close to the 22 grade

If the pitches have grades 22,22,22,22,22 the the CPR will be close to 23

The first scenario is fine. The second scenario could be fine as long as this is what the community want. 

There are some of 5 pitch routes which I have done in three. 

At some point we might bring in a floating grade or a indicate on how hard a route is for the grade. In which case this would solve the bump anyway.

Let's leave this open, but with the view it is a long term issue.
